Makefile

```diff
@@ -0,0 +1,11 @@
+build-k8s-helm-image:
+	docker build -t k8s-helm-deploy-image:latest -f eq-k8s-helm-deploy-image/Dockerfile .
```

Comment:
Very minor (feel free to ignore!) - the `-f` flag can be omitted for each of these as Docker will implicitly look for `Dockerfile` in the path, e.g.

```
docker build ./eq-k8s-helm-deploy-image -t k8s-helm-deploy-image:latest
```

Reply:
That's a nice one, only one concern is that, if I do that for all the commands, it doesn't work for the tinyproxy command and I have to alter the Dockerfile by updating the path of the tinyproxy.conf file. With this change, the Make command works, but the pipeline won't be happy with this new Dockerfile as it won't be able to find the tinyproxy.conf as the location would be different. 

I'm happy to change 3 of the Make commands and keep the tinyproxy the same, but I guess for consistency would it be best to keep it the same?

Reply:
Ah fair enough 👍  It's clear what the commands are doing in the `Makefile`, so happy to keep it as is